### ISSUE TITLE
travelapprovals@nodejs.org and thisdot not receiving my emails?

### ISSUE THREAD

**Issue by devsnek** [nodejs/admin#290]
I sent in my expense report a few months ago and never heard back, and in the last few days a thisdot employee emailed me but has since not responded to my communications. Would it be possible to somehow get in contact with them to see if my emails were categorized as spam or something?

**Comment by chowdhurian** [comment]
cc @@SofiEstevez

**Comment by benjamingr** [comment]
I assume they were not processed / dealt with yet - I received a "we got your email" for a report I sent on October 1st and no further response. I assumed it's just taking time.

**Comment by devsnek** [comment]
i haven't gotten any response, including "we got your email"

**Comment by devsnek** [comment]
well then

**Comment by mhdawson** [comment]
@ladyleet can you chase up an answer, even if its just "investigating"

**Comment by brianwarner** [comment]
Hi all, I can provide an update.  The travel fund reimbursement process was one of multiple things which transitioned to me when I picked up the project late last fall.  Once somebody is approved, I am responsible for verifying we have all the documents in order, and that it agrees with the amount in the markdown file.

There was a long backlog of items which needed to be addressed as part of moving onto the project.  One of those items was validating which payments had been issued, which were accounted for in the markdown file, and what needed to be corrected or updated.  This is really close to being fully done, and I think I'm in contact with everybody who has responded.

If for whatever reason you submitted and have not heard back, or if you want me to check on a payment status, please email me at travelapprovals@nodejs.org.

So, long story short, we had some hiccups but I think we're back on track.

**Comment by devsnek** [comment]
@benjamingr yep 👍
